css-positioning/absolute.css

```diff
@@ -0,0 +1,46 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 5em;
```

Comment:
Woah, that's a big movement! This is supposed to be on the right edge, yes? If so, moving it from the left edge seems like a lot of work 🤔